doc/building.md

```diff
@@ -139,3 +139,16 @@ to create a dummy X server, the server will be terminated when the test is
 complete.
 
 The logic for using VNC is encapsulated within the script _make/display.sh_.
+
+# Contribution guidelines
+
+Contributors are encouraged to follow code style conventions in [Java Style Guidelines](https://cr.openjdk.org/~alundblad/styleguide/index-v6.html) 
+where reasonable. jtreg's command-line flags have a certain style due to their
+age, but new flags should strive to follow [JEP 293: Guidelines for JDK Command-Line Tool Options](https://openjdk.org/jeps/293). 
+One notable exception is the rule about case-sensitivity - jtreg's command line
+flags are case insensitive and due to backwards compatibility it seems 
+reasonable to keep this going forward.
```

Comment:
Generally, it is good practice to not use time-dependent text, like "going forward", in any spec. 

I would replace the entire 148-150 lines with,
```
For backwards compatibility, `jtreg` option names are case-insensitive.
```

Reply:
Good advice, I've adopted this language.